### PR TITLE
Add potplayer

### PR DESCRIPTION
**Pull request by Aceralon**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/403)

### CODE REVIEW

**Comment by jazzdelightsme** [comment]
See also #439.

File: manifests/Daum/Potplayer/200512.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Publisher: Daum
+License: Daum PotPlayer End User License Agreement
+LicenseUrl: https://potplayer.daum.net/publicRelation
+AppMoniker: potplayer
+Tags: Media, media player, player, video, playback, potplayer, daum
+Homepage: https://potplayer.daum.net/
+Installers:
+  - Arch: x64
+    Url: https://t1.daumcdn.net/potplayer/PotPlayer/Version/200512/PotPlayerSetup64.exe
+    Sha256: FA1EEDE7A6383AAE08A9B45A9F19DA1C548606AA260B217B4B71EE4F26644138
```
Comment (OfficialEsco) [inline manifests/Daum/Potplayer/200512.yaml:13]: Dunno where you got that Sha from, but this is the correct one
```suggestion
    Sha256: cd184dda2686458726a1ec922ac10ee4ed0bb8112b29f256b84c272d12e1fd5a
```

File: manifests/Daum/Potplayer/200512.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    InstallerType: exe
+
```
Comment (OfficialEsco) [inline manifests/Daum/Potplayer/200512.yaml:15]:
```suggestion
Switches:
      Silent: /S
      SilentWithProgress: /S
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
PotPlayer will probably not get accepted because its Bundleware

**Pull request closed without merging**